docs/units-and-global-variables.rst

```diff
@@ -310,3 +310,12 @@ available for a contract type ``C``:
     regular calls.
     The same restrictions as with ``.creationCode`` also apply for this
     property.
+
+In addition to the properties above, the following properties are available
+for an interface type ``I``:
+
+``type(I).interfaceId``:
+    This property returns the `EIP-165 <https://eips.ethereum.org/EIPS/eip-165>`_
```

Comment:
Sorry to comment here again: Strictly, only functions return values and this is not a function.
Maybe
`A ``bytes4`` value containing the `EIP-165..`
and then you can remove the last sentence in the paragraph.